### PR TITLE
feat: [#188117010] add above license status field in cms for calendar…

### PR DESCRIPTION
**Pull request by FarazA22**
… events and reinstatements

### Ticket

https://www.pivotaltracker.com/story/show/188117010

### Approach
Add a field in the cms that will allow the content team to start adding content for an upcoming change to the license calendar and reinstatement events
I updated the preview to allow content to see the content.
All calendar events and reinstatements have been updated to an empty string

## Code author checklist

- [x] I have rebased this branch from the latest main branch
- [x] I have performed a self-review of my code
- [x] I have created and/or updated relevant documentation (EX: [Engineering Reference/FAQ](https://docs.google.com/document/d/1X4iWSGmBZdHYZ0jGqkwTR3_yyyqI_TiiJptfc8pi9Po)), if necessary
- [x] I have not used any relative imports
- [x] I have checked for and removed instances of unused config from CMS
- [x] I have pruned any instances of unused code
- [x] I have not added any markdown to labels, titles and button text in config
- [x] If I added/updated any values in `userData` (including `profileData`, `formationData` etc), then I added a new migration file
- [x] If I added any new collections to the CMS config, then I updated the search tool and `cmsCollections.ts` (see [CMS Additions in Engineering Reference/FAQ](https://docs.google.com/document/d/1X4iWSGmBZdHYZ0jGqkwTR3_yyyqI_TiiJptfc8pi9Po/edit#heading=h.fu5jdsrcqxbh))
- [x] I have updated relevant `.env` values in both `.env-template` and in Bitwarden

### CODE REVIEW

File: web/public/mgmt/config.yml [hunk 1]
```diff
@@ -3090,16 +3090,22 @@ collections:
...
           name: "expirationEventDisplayName",
           widget: "string",
           required: true,
         }
+      - { label: "Renewal Event Label", name: "renewalEventDisplayName", widget: "string", required: true }
+      - {
+          label: "Above License Status Content",
+          name: "aboveLicenseStatusContent",
+          widget: "markdown",
+          required: true,
```
Comment (jphechter) [inline web/public/mgmt/config.yml:3105]: Should this be `false`? Will every instance have content here?

File: web/public/mgmt/config.yml [hunk 2]
```diff
@@ -3505,8 +3511,14 @@ collections:
         value_field: "anytimeActionIcons.*.filename"
         display_fields: ["anytimeActionIcons.*.name"]
       - { label: "Url Slug - Internal Routing", name: "urlSlug", widget: "nospace", required: true }
+      - {
+          label: "Above License Status Content",
+          name: "aboveLicenseStatusContent",
+          widget: "markdown",
+          required: true,
```
Comment (jphechter) [inline web/public/mgmt/config.yml:3518]: Same question here

File: web/public/mgmt/config.yml (hunk 1 above)
Comment (FarazA22) [reply web/public/mgmt/config.yml:3105]: Yes, based on the designs, there is a license status card that would go in between the content. Based on the discussions we have had so far, it is my understanding they will always be content above and below the card.

Comment (FarazA22) [reply web/public/mgmt/config.yml:3105]: https://www.figma.com/design/Xr5I96TLqdoE6ZAorDTUUi/56-60-Business-Experience-Sprint-Designs?node-id=40784-46405&t=PEBE0BFJICY0t4m3-1

**Review by jphechter — APPROVED** [review]

**Review by aadedejifearless — APPROVED** [review]
LGTM

**Pull request merged**